causalpy/pymc_models.py

```diff
@@ -1,3 +1,14 @@
+"""
+Defines generic PyMC ModelBuilder class and subclasses for
+
+- WeightedSumFitter model for Synthetic Control experiments
+- LinearRegression model
+
+Models are intended to be used from inside an experiment
+class (see pymc_experiments.py). This is why the examples require some extra
```

Comment:
Could we change this to `(see :code:`pymc_experiments.py`)`. Or even better, link to the `causalpy.pymc_experiments` API page.